### PR TITLE
feat: Add new sample for Media CDN

### PR DESCRIPTION
**Pull request by msampathkumar**
Add path component for Media CDN Dual Token.

## Description

Fixes: http://b/269980260

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by kweinmeister** [comment]
@msampathkumar this is passing all tests; anything else required?

**Comment by msampathkumar** [comment]
Hi @kweinmeister thanks for the prompt! This is for some specific customer usecase, more details are at b/269980260

**Comment by msampathkumar** [comment]
@mengxin891029 Thank you for the comments.

Update: Changed the status of https://github.com/GoogleCloudPlatform/python-docs-samples/pull/10162(this request), from Draft to PR

**Review by kweinmeister — APPROVED** [review]

**Review by rsamborski — APPROVED** [review]

File: media_cdn/dualtoken.py [hunk 1]
```diff
@@ -173,3 +173,45 @@ def sign_token(
...
+
+    policy_pattern = u'{url_prefix}edge-cache-token=Expires={expires}&KeyName={key_name}'
+    policy = policy_pattern.format(
+        url_prefix=url_prefix,
+        expires=int(expiration_duration.total_seconds()),
+        key_name=key_name)
+
+    digest = ed25519.Ed25519PrivateKey.from_private_bytes(
+        decoded_key).sign(policy.encode('utf-8'))
+    signature = base64.urlsafe_b64encode(digest).decode('utf-8')
```
Comment (justin-mp) [inline media_cdn/dualtoken.py:209]: In the other functions you use `base64_encoder` to strip off the trailing padding.  You should probably do the same thing here.

**Pull request merged**